cerberus/validator.py

```diff
@@ -1171,7 +1171,14 @@ def _validate_excludes(self, excluded_fields, field, value):
 
                 self._unrequired_by_excludes.add(excluded_field)
 
-        if any(excluded_field in self.document for excluded_field in excluded_fields):
+        def _ignore_field(field):
+            # If the field is None and ignore_none_values is set, ignore it.
+            return self.document[field] is None and self.ignore_none_values
+
+        if any(
+            excluded_field in self.document and not _ignore_field(excluded_field)
```

Comment:
this function call in a loop is suboptimal performance-wise, and i see not much gain in expressiveness here.